packages/components/src/components/hive-navigation/index.tsx

```diff
@@ -83,7 +85,7 @@ export function HiveNavigation({
     },
   ],
   developerMenu,
-  searchProps,
+  search = <Search />,
```

Comment:
You moved the styles here to a .css file, right?

Reply:
yes